src/schema/index.ts

```diff
@@ -50,6 +50,36 @@ const Schema = gql`
       user: ID
     ): Profile
   }
-`
+  type ratingSystem {
+    name:String!
+    grade:[Int]!
+    description:[String]!
+    percentage:[String]!
+  }
+  type Mutation {
+    createRatingSystem(name: String!, grade:[Int]!, description: [String]!, percentage: [String]!): ratingSystem!
+  }
+  type Query{
+    getRatingSystem:[ratingSystem]
+  }
+  type grades {
```

Comment:
What is the use of these types you have defined here. I am unable to locate anywhere you have used them.
Remove them if they are not being used